features/offer_conditions.feature

```diff
@@ -0,0 +1,64 @@
+      | X123        | 10M           | U80             |
+
+  Scenario Outline: adding conditions - who can do it and when
+    Providers can add conditions to an application as part of making the offer,
+    or when the application has the 'conditional offer' status. Both the accredited body
+    and the non-accredited body can add conditions.
+
+    When an application has been made to a course X123
+    And the application in "<Application status>" state
+    Then a provider with code "<Provider code>" is able to add conditions: "<Can add conditions?>"
+
+    Examples:
+      | Provider code | Application status   | Can add conditions? | Notes                           |
+      | 10M           | application complete | N                   | Application in the wrong status |
+      | U80           | meeting conditions   | N                   | Application in the wrong status |
+      | U80           | unconditional offer  | N                   | Application in the wrong status |
+      | S13           | conditional offer    | N                   | Wrong provider                  |
+      | U80           | conditional offer    | Y                   |                                 |
+      | 10M           | conditional offer    | Y                   |                                 |
+
+  Scenario Outline: amending conditions - who can do it and when
+    Once a provider makes a conditional offer, they can amend these conditions,
+    but only if the candidate has not accepted their offer.
+
+    Given an application has been made to a course X123
+    And the application in "<Application status>" state
+    Then a provider with code "<provider code>" is able to amend conditions: "<Can amend conditions?>"
+
+    Examples:
+      | provider code | Application status   | Can amend conditions? | Notes                           |
+      | 10M           | application complete | N                     | Application in the wrong status |
+      | U80           | meeting conditions   | N                     | Application in the wrong status |
+      | U80           | unconditional offer  | N                     | Application in the wrong status |
+      | S13           | conditional offer    | N                     | Wrong provider                  |
+      | U80           | conditional offer    | Y                     |                                 |
+      | 10M           | conditional offer    | Y                     |                                 |
+
+  Scenario: amending condition changes the offer's expiry time
+    The expiry time on the offer is reset when conditions are successfully amended.
+
+    Given an application has been made to a course X123
+    And the application in "conditional offer" state
+    And the expiry time on the offer is "12 June 2019 12:00:00 PM"
+    When the provider with code "U80" amends a condition at 8:00 AM on 13 June 2019
+    And the new expiry time on the offer is "<expected DBD time>"
```

Comment:
I don't think "expected DBD time" is defined anywhere?

Reply:
Added in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/189/commits/838db2f0ee2aab7c4005639b61d4f43dcddcb490